src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisEvent.scala

```diff
@@ -45,6 +46,59 @@ case class Location(
   }
 
   override def hashCode(): Int = wofId.hashCode
+
+  override def compare(that: Location): Int = {
+    Location.layerToInt(layer).compare(Location.layerToInt(that.layer))
+  }
+}
+
+object Location {
+  /**
+   * @see https://github.com/whosonfirst/whosonfirst-placetypes
+   */
+  def layerToInt(layer: String): Int = {
+    if ("continent".equalsIgnoreCase(layer)) {
```

Comment:
Can you lowercase `layer` first and then do a match expression here?

Reply:
a73d5f7